src/get_table_metrics.py

```diff
@@ -136,6 +141,8 @@ def main(argv: Sequence[str]) -> int:
             'TableName',
             'TableSizeBytes',
             'ItemCount',
+            'GlobalSecondaryIndexCount',
+            'LocalSecondaryIndexCount',
```

Comment:
Can we move the new columns to the last? I think it's a bit more consistent that way?